salt_resources/states/network/init.sls

```diff
@@ -0,0 +1,6 @@
+system:
+  network.system:
```

Comment:
deep system interaction like this are somewhat unstable in saltstack. it frustratingly enough often necessary to stick to simple sls states, like cli calls and file editing.

we can try using the network.system